src/System.Net.Primitives/tests/FunctionalTests/IPAddressTest.cs

```diff
@@ -242,23 +247,55 @@ public static void Equals_Compare_Success()
             Assert.False(ip4.GetHashCode().Equals(ip6.GetHashCode()));
         }
 
+        [Theory]
+        [MemberData(nameof(GetValidIPAddresses))]
+        public static void GetHashCode_ValidIPAddresses_Success(IPAddress ipAddress)
+        {
+            ipAddress.GetHashCode();
```

Comment:
I suppose there is no value in also calling this twice and checking it's equals. I see you do that above for a couple values.

Reply:
I think there's value in that.